packet/flatcar-linux/kubernetes/workers/cl/install.yaml.tmpl

```diff
@@ -0,0 +1,81 @@
+---
+systemd:
+  units:
+    - name: installer.service
+      enable: true
+      contents: |
+        [Unit]
+        StartLimitBurst=5
+        StartLimitIntervalSec=3600s
+        Requires=network-online.target
+        After=network-online.target
+        [Service]
+        Type=simple
+        Restart=always
+        RestartSec=60
+        ExecStart=/opt/installer
+        [Install]
+        WantedBy=multi-user.target
+    # Avoid using the standard SSH port so terraform apply cannot SSH until
+    # post-install. But admins may SSH to debug disk install problems.
+    # After install, sshd will use port 22 and users/terraform can connect.
+    - name: sshd.socket
+      dropins:
+        - name: 10-sshd-port.conf
+          contents: |
+            [Socket]
+            ListenStream=
+            ListenStream=2222
+storage:
+  files:
+    - path: /opt/postinstall-ignition.json
+      filesystem: root
+      mode: 0500
+      contents:
+        inline: |
+          ${postinstall_ignition}
+    - path: /opt/installer
+      filesystem: root
+      mode: 0500
+      contents:
+        inline: |
+          #!/bin/bash -ex
+
+          # A comma-separated list of major device numbers. Modify to control which device types
+          # are considered for OS installation and data RAID.
+          # https://www.kernel.org/doc/Documentation/admin-guide/devices.txt
+          major_numbers="8,259"
+
+          # This function returns the path to the block device which represents the smallest disk
+          # attached to the system. The output can be passed to the flatcar-install script.
+          function select_install_disk() {
+            local major_numbers="$1"
+
+            local disk=$(lsblk -lnpd -I "$${major_numbers}" \
+              | sort -h -k 4,4 \
+              | head -1 \
+              | awk '{print $1}'
+            )
+
+            echo "$${disk}"
+          }
+
+          os_disk="$(select_install_disk $${major_numbers})"
```

Comment:
This ignores the `setup_raid` variable, so those features won't be available on arm without warning

Reply:
The variable is passed to the second ignition and on first boot of the installed OS the RAID will be created as with the native Packet installation.